cmd/snap-confine/snap-confine.c

```diff
@@ -112,6 +112,10 @@ int main(int argc, char **argv)
 #endif				// ifdef HAVE_SECCOMP
 
 	if (geteuid() == 0) {
+		// ensure that "/" or "/snap" is mounted with the
```

Comment:
With the locking helper in #3149 we should be able to achieve this trivially now.